File: Jack.Model/DTOs/FamiliaPresenca.vb
<Serializable()> _
Public Class FamiliaPresenca

    Public Sub New()

        Nivel = 0
        Familia = String.Empty
        Data = New DateTime()

    End Sub


    Public Sub New(strFamilia As String, intNivel As Integer, DataReuniao As DateTime)

        Nivel = intNivel
        Familia = strFamilia
        Data = DataReuniao

    End Sub

    Public Property Familia As String
    Public Property Nivel As Integer
    Public Property Data As DateTime

    Public ReadOnly Property DataFormated As String
        Get
            Return Data.Day.ToString("00") + "/" + Data.Month.ToString("00") + "/" + Data.Year.ToString("0000")
        End Get
    End Property




End Class

File: Jack.Model/DTOs/FamiliaPresenca.cs
using System;

namespace Jack.Model
{
    [Serializable()]
    public class FamiliaPresenca
    {


        public FamiliaPresenca()
        {
            Nivel = 0;
            Familia = string.Empty;
            Data = new DateTime();

        }



        public FamiliaPresenca(string strFamilia, int intNivel, DateTime DataReuniao)
        {
            Nivel = intNivel;
            Familia = strFamilia;
            Data = DataReuniao;

        }

        public string Familia { get; set; }
        public int Nivel { get; set; }
        public DateTime Data { get; set; }

        public string DataFormated
        {
            get { return Data.Day.ToString("00") + "/" + Data.Month.ToString("00") + "/" + Data.Year.ToString("0000"); }
        }

    }
}
